### PR TITLE
Assert rather than returning from BigQuery schema assertion method

### PR DESCRIPTION
**Pull request by cdzombak**
This fixes the method introduced in #22.

### CODE REVIEW

**Comment by zakird** [comment]
Before merging this, I'd like to clarify whether this function should exist instead of using ```json.dumps(a, sort_keys=True) == json.dumps(b, sort_keys=True)```

File: zschema/tests.py [hunk 1]
```diff
@@ -127,8 +127,7 @@ def assertBigQuerySchemaEqual(self, a, b):
         BigQuery schemas consist of lists whose order doesn't matter, dicts,
         and privimites.
         """
-        if type(a) is not type(b):
-            return False
+        self.assertEquals(type(a), type(b))
```
Comment (zakird) [inline zschema/tests.py]: I'm still also worrisome that this logic is un-pythonic and apt to break, for example in different cases of `basestring`.

**Comment by cdzombak** [comment]
> Before merging this, I'd like to clarify whether this function should exist instead of using json.dumps(a, sort_keys=True) == json.dumps(b, sort_keys=True)

Replacing this assertion method with one based on `json.dumps` results in a test failure, because the failure here has been with _lists_ that have different orders. `json.dumps` does not dump lists in a consistent order, regardless of `sort_keys`.

**Comment by cdzombak** [comment]
> I'm still also worried that this logic is un-pythonic and apt to break, for example in different cases of basestring.

BigQuery schemas are currently JSON with strings in them. IMO, if this changes in the future we can change this test to reflect that new reality.

**Pull request merged**